Exiled.API/Features/Roles/Scp049Role.cs

```diff
@@ -122,6 +150,173 @@ public float GoodSenseCooldown
             }
         }
 
+        /// <summary>
+        /// Gets or sets the amount of time before SCP-049 can attack again.
+        /// </summary>
+        public float AttackCooldown
+        {
+            get => AttackAbility.Cooldown.Remaining;
+            set
+            {
+                AttackAbility.Cooldown.Remaining = value;
+                AttackAbility.ServerSendRpc(true);
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the duration of the <see cref="Scp049CallAbility"/>.
+        /// </summary>
+        public float CallDuration
+        {
+            get => CallAbility.Duration.Remaining;
+            set
+            {
+                CallAbility.Duration.Remaining = value;
+                CallAbility.ServerSendRpc(true);
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the duration of the <see cref="Scp049SenseAbility"/>.
+        /// </summary>
+        public float GoodSenseDuration
+        {
+            get => SenseAbility.Duration.Remaining;
+            set
+            {
+                SenseAbility.Duration.Remaining = value;
+                SenseAbility.ServerSendRpc(true);
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the distance of the Sense Ability.
+        /// </summary>
+        public float SenseDistance
+        {
+            get => SenseAbility._distanceThreshold;
+            set => SenseAbility._distanceThreshold = value;
+        }
+
+        /// <summary>
+        /// Gets the <see cref="Scp049GameRole"/> instance.
+        /// </summary>
+        protected Scp049GameRole Internal { get; }
+
+        /// <summary>
+        /// Lose the actual target of the SCP-049 Sense Ability.
```

Comment:
Lose the current target of the Good Sense ability.

Reply:
done